tests/e2e/ContainerFile.control

```diff
@@ -41,7 +43,6 @@ RUN dnf install -y \
 
 WORKDIR /root
 
-RUN dnf -y copr enable rhcontainerbot/qm centos-stream-9
```

Comment:
@Yarboa Thanks for the patch. I am wondering if moving this line above helps or just adding iptables-services as package needed to be installed is enough?

Reply:
Could you please double check/test?

Reply:
Hi @dougsland 
Podman is installed from this repo

```
podman exec -it control /bin/bash -c "dnf info podman" | grep Repository
Repository   : @System
Repository   : copr:copr.fedorainfracloud.org:rhcontainerbot:qm
```
Otherwise, I got podman 4.4.1
I will check it, again

Reply:
Ack

Reply:
Verified, it is working on clean image